src/Material.cpp

```diff
@@ -1394,28 +1533,17 @@ void Material::printString() {
  *          subclasses can break up loops over energy groups in such a way
  *          to "expose" the SIMD nature of the algorithm.
  */
-void Material::alignData() {
+void MacroMaterial::alignData() {
 
   /* If the data has already been aligned, do nothing */
   if (_data_aligned)
     return;
-
-  if (_num_groups <= 0)
-    log_printf(ERROR, "Unable to align Material %d data since the "
-               "cross-sections have not yet been set\n", _id);
-
-  _num_vector_groups = (_num_groups / VEC_LENGTH) + 1;
+    
+  Material::alignData();
```

Comment:
It doesn't appear that this routine is doing what it is intended to do. Am I missing something?